Application/Bin/.gitignore

```diff
@@ -1,2 +1,3 @@
 *.user*
-snapshot_images/
\ No newline at end of file
+exported_results/
```

Comment:
Specify that it's both exports and snapshots. Also capitalize & add spacing

eg. "Snapshots & Exports", "Exports & Snapshots", "Exports and Snapshots", etc.